packages/dds/merge-tree/src/client.ts

```diff
@@ -853,20 +936,176 @@ export class Client extends TypedEventEmitter<IClientEvents> {
 			segmentGroup === NACKedSegmentGroup,
 			0x034 /* "Segment group not at head of pending rebase queue" */,
 		);
+		assert(
+			segmentGroup.localSeq !== undefined,
```

Comment:
nothing to do here, but we should make this required, rather than needing to assert.